dask/bytes/s3.py

```diff
@@ -33,15 +33,13 @@ def __init__(self, key=None, username=None, secret=None, password=None,
             secret = password
         if secret is not None:
             kwargs['secret'] = secret
-        self.blocksize = blocksize
```

Comment:
Previously, there was a `blocksize` keyword to the DaskS3FileSystem constructor, which was meant to achieve the same thing: always call `open()` (below) with the stated block size value. However, there was no way to pass in the value from read_csv (or read_parquet). Instead, we now pass a set of kwargs - the non-credentials part of storage_options - through to the upstream constructor.